include/phasar/PhasarLLVM/DataFlowSolver/IfdsIde/Problems/IDEInstInteractionAnalysis.h

```diff
@@ -147,30 +150,32 @@ class IDEInstInteractionAnalysisT
       return std::make_shared<Gen<d_t>>(Alloca, this->getZeroValue());
     }
 
-    // Handle indirect taints, i. e., propagate values that depend on branch
-    // conditions whose operands are tainted.
-    if (EnableIndirectTaints) {
-      if (auto br = llvm::dyn_cast<llvm::BranchInst>(curr);
-          br && br->isConditional()) {
-        return std::make_shared<LambdaFlow<d_t>>([=](d_t src) {
-          container_type ret = {src, br};
-          if (src == br->getCondition()) {
-            for (auto succ : br->successors()) {
-              // this->indirecrTaints[succ].insert(src);
-              for (auto &inst : succ->instructionsWithoutDebug()) {
-                ret.insert(&inst);
-              }
-            }
-          }
-          return ret;
-        });
-      }
-    }
+    // // Handle indirect taints, i. e., propagate values that depend on branch
+    // // conditions whose operands are tainted.
+    // if (EnableIndirectTaints) {
+    //   if (auto br = llvm::dyn_cast<llvm::BranchInst>(curr);
+    //       br && br->isConditional()) {
+    //     return std::make_shared<LambdaFlow<d_t>>([=](d_t src) {
+    //       container_type ret = {src, br};
+    //       if (src == br->getCondition()) {
+    //         for (auto succ : br->successors()) {
+    //           // this->indirecrTaints[succ].insert(src);
+    //           for (auto &inst : succ->instructionsWithoutDebug()) {
+    //             ret.insert(&inst);
+    //           }
+    //         }
+    //       }
+    //       return ret;
+    //     });
+    //   }
+    // }
```

Comment:
Hmm, I think this should be readded or am I wrong? Or do we handle the propagation of indirect taints somewhere else?

Reply:
Added it again. There is also a bug in the code which I fixed on the fly.